Static/AnalysisLibrary.hs

```diff
@@ -269,8 +271,10 @@ anaLibDefn lgraph opts topLns libenv dg (Lib_defn ln alibItems pos ans) file
       ([], dg { globalAnnos = allAnnos }, libenv
       , lgraph, Map.empty) (map item alibItems)
   let dg1 = computeDGraphTheories libenv' $ markFree libenv'
-        $ markHiding libenv' $ fromMaybe dg' $ maybeResult
-        $ shortcutUnions dg'
+        $ markHiding libenv' 
+        $ fromMaybe dg' $ maybeResult
+        $ shortcutUnions 
+        dg'
```

Comment:
this is a mere reformatting. Is this needed?

Reply:
It's not, it was caused by me trying to find out how much I can comment. I have reverted it.